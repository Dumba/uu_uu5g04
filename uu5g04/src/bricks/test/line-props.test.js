/**
 * Copyright (C) 2019 Unicorn a.s.
 * 
 * This program is free software; you can use it under the terms of the UAF Open License v01 or
 * any later version. The text of the license is available in the file LICENSE or at www.unicorn.com.
 * 
 * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even
 * the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See LICENSE for more details.
 * 
 * You may contact Unicorn a.s. at address: V Kapslovne 2767/2, Praha 3, Czech Republic or
 * at the email: info@unicorn.com.
 */

import React from 'react';
import {shallow} from 'enzyme';
import UU5 from "uu5g04";
import "uu5g04-bricks";
import enzymeToJson from 'enzyme-to-json';
import TestTools from "../../core/test/test-tools.js";


const TagName = "UU5.Bricks.Line";

const CONFIG = {
  mixins: [
    "UU5.Common.BaseMixin",
    "UU5.Common.ElementaryMixin",
    "UU5.Common.ColorSchemaMixin",
    "UU5.Common.NestingLevelMixin",
    "UU5.Common.PureRenderMixin"

  ],
  props: {
    size: {
      values: ["s", "m", "l", "xl"]
    }
  },
  requiredProps: {},
  opt: {
    shallowOpt: {
      disableLifecycleMethods: false
    },
    enzymeToJson: false
  }
};


describe(`${TagName}`, () => {
  TestTools.testProperties(TagName, CONFIG);
});

describe(`${TagName} docKit examples`, () => {
  it(`${TagName} should render without crash`, () => {
    const wrapper = shallow(
      <UU5.Bricks.Container id={"iiROOT"}>
        <UU5.Bricks.Paragraph id={"uuID01"}/>
        <UU5.Bricks.Line id={"uuID02"} colorSchema='primary' size='s'/>
      <UU5.Bricks.Paragraph />
        <UU5.Bricks.Line id={"uuID03"} colorSchema='green-rich' size='m'/>
        <UU5.Bricks.Paragraph id={"uuID04"}/>
        <UU5.Bricks.Line id={"uuID05"} colorSchema='warning' size='l'/>
        <UU5.Bricks.Paragraph id={"uuID06"}/>
        <UU5.Bricks.Line id={"uuID07"} colorSchema='red-rich' size='xl'/>
      </UU5.Bricks.Container>
    );
    expect(enzymeToJson(wrapper)).toMatchSnapshot();
  });
});










