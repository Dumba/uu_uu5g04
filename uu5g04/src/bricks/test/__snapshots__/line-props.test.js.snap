// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UU5.Bricks.Line className 1`] = `
<div
  className="uu5-bricks-line uu5-common-bg my-classname uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 1`] = `
<div
  className="color-schema-default uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 2`] = `
<div
  className="color-schema-red uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 3`] = `
<div
  className="color-schema-red uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 4`] = `
<div
  className="color-schema-red-rich uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 5`] = `
<div
  className="color-schema-red-rich uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 6`] = `
<div
  className="color-schema-green uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 7`] = `
<div
  className="color-schema-green uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 8`] = `
<div
  className="color-schema-green-rich uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 9`] = `
<div
  className="color-schema-green-rich uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 10`] = `
<div
  className="color-schema-yellow uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 11`] = `
<div
  className="color-schema-yellow uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 12`] = `
<div
  className="color-schema-yellow-rich uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 13`] = `
<div
  className="color-schema-yellow-rich uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 14`] = `
<div
  className="color-schema-purple uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 15`] = `
<div
  className="color-schema-purple uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 16`] = `
<div
  className="color-schema-purple-rich uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 17`] = `
<div
  className="color-schema-purple-rich uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 18`] = `
<div
  className="color-schema-brown uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 19`] = `
<div
  className="color-schema-brown uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 20`] = `
<div
  className="color-schema-brown-rich uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 21`] = `
<div
  className="color-schema-brown-rich uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 22`] = `
<div
  className="color-schema-orange uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 23`] = `
<div
  className="color-schema-orange uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 24`] = `
<div
  className="color-schema-orange-rich uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 25`] = `
<div
  className="color-schema-orange-rich uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 26`] = `
<div
  className="color-schema-cyan uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 27`] = `
<div
  className="color-schema-cyan uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 28`] = `
<div
  className="color-schema-cyan-rich uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 29`] = `
<div
  className="color-schema-cyan-rich uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 30`] = `
<div
  className="color-schema-grey uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 31`] = `
<div
  className="color-schema-grey uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 32`] = `
<div
  className="color-schema-grey-rich uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 33`] = `
<div
  className="color-schema-grey-rich uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 34`] = `
<div
  className="color-schema-pink uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 35`] = `
<div
  className="color-schema-pink uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 36`] = `
<div
  className="color-schema-pink-rich uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 37`] = `
<div
  className="color-schema-pink-rich uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line colorSchema 38`] = `
<div
  className="color-schema-pink uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line controlled 1`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line controlled 2`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line controlled 3`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line controlled 4`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line default props 1`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line disabled 1`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-common-disabled uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
>
  <span
    className="uu5-common-disabled-cover"
    key="disabledCover"
  />
</div>
`;

exports[`UU5.Bricks.Line disabled 2`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line disabled 3`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line disabled 4`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-common-disabled uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
>
  <span
    className="uu5-common-disabled-cover"
    key="disabledCover"
  />
</div>
`;

exports[`UU5.Bricks.Line docKit examples UU5.Bricks.Line should render without crash 1`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="iiROOT"
  style={Object {}}
  title={null}
>
  <Paragraph
    className={null}
    content={null}
    controlled={true}
    disabled={false}
    dynamic={false}
    hidden={false}
    id="uuID01"
    ignoreInnerHTML={false}
    key="uuID01"
    mainAttrs={null}
    name={null}
    nestingLevel={null}
    noIndex={false}
    pureRender={false}
    ref_={null}
    selected={false}
    style={null}
    tooltip={null}
  />
  <Line
    borderRadius={null}
    className={null}
    colorSchema="primary"
    controlled={true}
    disabled={false}
    hidden={false}
    id="uuID02"
    key="uuID02"
    mainAttrs={null}
    name={null}
    nestingLevel={null}
    noIndex={false}
    pureRender={false}
    ref_={null}
    selected={false}
    size="s"
    style={null}
    tooltip={null}
    vertical={false}
  />
  <Paragraph
    className={null}
    content={null}
    controlled={true}
    disabled={false}
    dynamic={false}
    hidden={false}
    id={null}
    ignoreInnerHTML={false}
    key="uu5-content-2"
    mainAttrs={null}
    name={null}
    nestingLevel={null}
    noIndex={false}
    pureRender={false}
    ref_={null}
    selected={false}
    style={null}
    tooltip={null}
  />
  <Line
    borderRadius={null}
    className={null}
    colorSchema="green-rich"
    controlled={true}
    disabled={false}
    hidden={false}
    id="uuID03"
    key="uuID03"
    mainAttrs={null}
    name={null}
    nestingLevel={null}
    noIndex={false}
    pureRender={false}
    ref_={null}
    selected={false}
    size="m"
    style={null}
    tooltip={null}
    vertical={false}
  />
  <Paragraph
    className={null}
    content={null}
    controlled={true}
    disabled={false}
    dynamic={false}
    hidden={false}
    id="uuID04"
    ignoreInnerHTML={false}
    key="uuID04"
    mainAttrs={null}
    name={null}
    nestingLevel={null}
    noIndex={false}
    pureRender={false}
    ref_={null}
    selected={false}
    style={null}
    tooltip={null}
  />
  <Line
    borderRadius={null}
    className={null}
    colorSchema="warning"
    controlled={true}
    disabled={false}
    hidden={false}
    id="uuID05"
    key="uuID05"
    mainAttrs={null}
    name={null}
    nestingLevel={null}
    noIndex={false}
    pureRender={false}
    ref_={null}
    selected={false}
    size="l"
    style={null}
    tooltip={null}
    vertical={false}
  />
  <Paragraph
    className={null}
    content={null}
    controlled={true}
    disabled={false}
    dynamic={false}
    hidden={false}
    id="uuID06"
    ignoreInnerHTML={false}
    key="uuID06"
    mainAttrs={null}
    name={null}
    nestingLevel={null}
    noIndex={false}
    pureRender={false}
    ref_={null}
    selected={false}
    style={null}
    tooltip={null}
  />
  <Line
    borderRadius={null}
    className={null}
    colorSchema="red-rich"
    controlled={true}
    disabled={false}
    hidden={false}
    id="uuID07"
    key="uuID07"
    mainAttrs={null}
    name={null}
    nestingLevel={null}
    noIndex={false}
    pureRender={false}
    ref_={null}
    selected={false}
    size="xl"
    style={null}
    tooltip={null}
    vertical={false}
  />
</div>
`;

exports[`UU5.Bricks.Line hidden 1`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-common-hidden uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line hidden 2`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line hidden 3`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line hidden 4`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-common-hidden uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line id 1`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="myId"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line mainAttrs 1`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line name 1`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  name="myName"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line nestingLevel 1`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line nestingLevel 2`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line nestingLevel 3`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line nestingLevel 4`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line nestingLevel 5`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line nestingLevel 6`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line nestingLevel 7`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line nestingLevel 8`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line nestingLevel 9`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line nestingLevel 10`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line nestingLevel 11`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line nestingLevel 12`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line nestingLevel 13`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line nestingLevel 14`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line nestingLevel 15`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line nestingLevel 16`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line nestingLevel 17`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line nestingLevel 18`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line nestingLevel 19`] = `""`;

exports[`UU5.Bricks.Line nestingLevel 20`] = `""`;

exports[`UU5.Bricks.Line noIndex 1`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-noindex uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line noIndex 2`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line noIndex 3`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line noIndex 4`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-noindex uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line pureRender 1`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line pureRender 2`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line pureRender 3`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line pureRender 4`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line selected 1`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-common-selected uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line selected 2`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line selected 3`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line selected 4`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-common-selected uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line size 1`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-s"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line size 2`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line size 3`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line size 4`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-l"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line size 5`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-l"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line size 6`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-xl"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line size 7`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-xl"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line size 8`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-l"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Line style 1`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={
    Object {
      "borderRadius": "2px",
      "padding": "8px",
    }
  }
  title={null}
/>
`;

exports[`UU5.Bricks.Line style 2`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={
    Object {
      "backgroundColor": "red",
      "color": "blue",
    }
  }
  title={null}
/>
`;

exports[`UU5.Bricks.Line style 3`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={
    Object {
      "backgroundColor": "red",
      "color": "blue",
    }
  }
  title={null}
/>
`;

exports[`UU5.Bricks.Line style 4`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={
    Object {
      "borderRadius": "2px",
      "padding": "8px",
    }
  }
  title={null}
/>
`;

exports[`UU5.Bricks.Line tooltip 1`] = `
<div
  className="uu5-bricks-line uu5-common-bg uu5-bricks-line-size-m"
  id="uuID"
  style={Object {}}
  title="My Tooltip"
/>
`;
